automation/taskcluster/lib/tasks.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         self.date = arrow.get(date_string)
         self.trust_level = trust_level
 
-    def craft_assemble_release_task(self, architectures, build_type, is_staging, version_name):
+    def craft_assemble_release_task(self, architectures, build_type, is_staging, version_name, run_coverage=False):
```

Comment:
Don't we only want to do coverage checking on non-release builds?

Reply:
Fixed.